msal/src/main/java/com/microsoft/identity/client/AuthenticationCallback.java

```diff
@@ -27,6 +27,7 @@
 import com.microsoft.identity.client.exception.MsalException;
 import com.microsoft.identity.client.exception.MsalServiceException;
 import com.microsoft.identity.client.exception.MsalUiRequiredException;
+import com.microsoft.identity.client.exception.MsalUserCancelException;
```

Comment:
MsalUserCancelException seems not being used.

Reply:
Fixed.  Thanks Heidi.  I also fixed the test app at the same time.